### ISSUE TITLE
CLI 'nano_node --wallet_create' or 'nano_node --wallet_list' commands don't work

### ISSUE THREAD

**Issue by aglkm** [nanocurrency/nano-node#2824]
Run ./nano_node --wallet_create command and you will get the following:
Assertion (!flags.read_only) failed
nano::node::node(boost::asio::io_context&, const boost::filesystem::path&, nano::alarm&, const nano::node_config&, nano::work_pool&, nano::node_flags, unsigned int)
/workspace/nano/node/node.cpp:368

 0# 0x00000000008BC363 in ./nano_node
 1# 0x00000000008BD604 in ./nano_node
 2# 0x000000000063531A in ./nano_node
 3# 0x00000000006372E8 in ./nano_node
 4# 0x000000000051F5AF in ./nano_node
 5# 0x0000000000523B3F in ./nano_node
 6# 0x0000000000480BB1 in ./nano_node
 7# __libc_start_main in /lib/x86_64-linux-gnu/libc.so.6
 8# 0x00000000004C5339 in ./nano_node

Aborted

**Comment by wezrule** [comment]
It seems this is an issue if the node hasn't been started yet (not properly initialised `data.ldb` file). Could you run with the `--daemon` flag first, wait for the command line output (with network etc) and try the command again please.

**Comment by aglkm** [comment]
OK, I've tested a bit.
If data.ldb file is located inside default home(~/Nano) directory there is no such issue.
However, in my scenario, I have started nano_node with custom directory(--data_path option), so in this case nano_node --wallet_create doesn't work.
I see that 'nano_node --wallet_create --data_path=...' works.
It would be nice to have a message saying something like "node requires data.ldb file for the operation but can't find it" in such case.
